### DISCUSSION TITLE
We are preparing v3.6.0 for a December release.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1898] [Announcements]
We are starting to prepare for the v3.6.0 release, which will be done early December.

Please remember v3.6.0 allows API changes, so if you have things waiting you want to be included in the library, now is the right time.

**Comment by MrWaloo** [comment]
Will there be any change to be done in the scripts written for pymodbus V3.5.4 or will this version be compatible?

**Reply by janiversen** [reply]
Look in API_CHANGES.rst when we do the release or in advance on the dev branch, it shows all API changes.

The rule is simple and used in the majority of projects. The version scheme is vX.Y.Z
- when Z changes there are no API changes, so its plug and run ready
- when Y changes there are normally API changes, so scripts need to be controlled
- when X changes there are major API/functional changes, so often scripts need to be rewritten.
